apps/sms/views/conversation/js/conversation.js

```diff
@@ -2248,18 +2244,25 @@ var ConversationView = {
     this.container.classList.remove('hide');
   },
 
-  // FIXME/bug 983411: phoneNumber not needed.
-  simSelectedCallback: function conv_simSelected(phoneNumber, cardIndex) {
```

Comment:
ha ha . sure :)  Julienw reminded me about multisim action thing. Will fix in other dedicated bug.